src/AutoRest.CSharp/Generation/Writers/RestClientWriter.cs

```diff
@@ -243,6 +244,23 @@ private void WriteRequestCreation(CodeWriter writer, RestClientMethod clientMeth
                             flattenedSchemaRequestBody.Serialization,
                             w => w.Append(modelVariable));
                         break;
+                    case UrlEncodedBody urlEncodedRequestBody:
+                        var urlContent = new CodeWriterDeclaration("content");
+
+                        WriteHeaders(writer, clientMethod, request, content: true);
+                        writer.Append($"var {urlContent:D} = new Azure.Core.FormUrlEncodedContent ();\n");
```

Comment:
sorry, missed one more thing.

```suggestion
                        writer.Line($"var {urlContent:D} = new {typeof(FormUrlEncodedContent)}();");
```